src/elements/content-sidebar/versions/VersionsItem.js

```diff
@@ -82,9 +85,18 @@ const VersionsItem = ({
     const versionInteger = versionNumber ? parseInt(versionNumber, 10) : 1;
     const versionTime = restoredAt || trashedAt || createdAt;
     const versionTimestamp = versionTime && new Date(versionTime).getTime();
-    const versionUserName = selectors.getVersionUser(version).name || (
-        <FormattedMessage {...messages.versionUserUnknown} />
-    );
+    const versionUser = selectors.getVersionUser(version);
+    let versionUserName = versionUser.name || <FormattedMessage {...messages.versionUserUnknown} />;
+    if (uploader_display_name) {
+        if (versionUser.id === PLACEHOLDER_USER.id) {
```

Comment:
Do these conditions need to be nested or can we make judicious use of `&&`?

Reply:
ooh yes, that can probably happen